sections/collapsible-content.liquid

```diff
@@ -0,0 +1,382 @@
+{{ 'component-accordion.css' | asset_url | stylesheet_tag }}
+{{ 'collapsible-content.css' | asset_url | stylesheet_tag }}
+
+<div class="collapsible-content layout-{{ section.settings.layout | split: '_' | first }}-container color-{{ section.settings.color_scheme }}{% if section.settings.layout == 'section_container' %} page-width{% endif %}">
+  <div class="collapsible-content-wrapper{% if section.settings.layout == 'section_container' %} color-{{ section.settings.container_color_scheme }}{% endif %}">
+    <div class="{% if section.settings.image == blank %} collapsible-content-wrapper--narrow{% else %} page-width{% endif%}">
+      <div style="text-align: {{ section.settings.heading_alignment }}">
+        {%- if section.settings.caption != blank -%}
+          <p class="caption-with-letter-spacing"> {{ section.settings.caption }}</p>
+        {% endif %}
+        {%- if section.settings.heading != blank -%}
+          <h2 class="h1">{{ section.settings.heading }}</h2>
+        {%- else -%}
+          <h2 class="visually-hidden">{{ 'accessibility.collapsible_content_title' | t }}</h2>
+        {% endif %}
+      </div>
+      <div class="grid grid--1-col grid--2-col-tablet collapsible-content__grid{% if section.settings.desktop_layout == 'text_first' %} collapsible-content__grid--reverse{% endif %}">
+        {%- if section.settings.image != blank -%}
+          <div class="grid__item grid__item--image">
+            <div class="collapsible-content__media collapsible-content__media--{{ section.settings.image_ratio }} media"
+              {% if section.settings.image_ratio == 'adapt' and section.settings.image != blank %} style="padding-bottom: {{ 1 | divided_by: section.settings.image.aspect_ratio | times: 100 }}%;"{% endif %}
+            >
+              <img
+                srcset="{%- if section.settings.image.width >= 165 -%}{{ section.settings.image | img_url: '165x' }} 165w,{%- endif -%}
+                  {%- if section.settings.image.width >= 360 -%}{{ section.settings.image | img_url: '360x' }} 360w,{%- endif -%}
+                  {%- if section.settings.image.width >= 535 -%}{{ section.settings.image | img_url: '535x' }} 535w,{%- endif -%}
+                  {%- if section.settings.image.width >= 750 -%}{{ section.settings.image | img_url: '750x' }} 750w,{%- endif -%}
+                  {%- if section.settings.image.width >= 1070 -%}{{ section.settings.image | img_url: '1070x' }} 1070w,{%- endif -%}
+                  {%- if section.settings.image.width >= 1500 -%}{{ section.settings.image | img_url: '1500x' }} 1500w,{%- endif -%}
+                  {{ section.settings.image | img_url: 'master' }} {{ section.settings.image.width }}w"
```

Comment:
I'd add a media size between `1070` and `1500` to close the gap between the two a bit and account for other screen sizes a bit better.